code/_onclick/hud/screen_objects.dm

```diff
@@ -678,9 +678,10 @@
 	..()
 	holder = C
 	holder.screen += src
-	var/titlescreen = TITLESCREEN
-	if(titlescreen)
-		icon_state = titlescreen
+
+	if(SStitle.title_screen)
+		icon_state = SStitle.title_screen.icon_state

```

Comment:
As an indestructible turf on z =2 its about as safe a bet as a turf gets. Using it as storage is a quick and easy way to make the splash screen match the end of round fade out.

Regardless I'd still be storing the turf either way because it saves a search for it later.